meeting-notes/20190108.md

```diff
@@ -0,0 +1,175 @@
+   - Started contributing to ipfs-log:
+     - Use IPLD: https://github.com/orbitdb/ipfs-log/pull/213
+     - Helped review other PRs related to improvements we discussed with ipfs-log developers
+ - In progress:
+     - Add timeouts to ipfs-log read & write functions: https://github.com/orbitdb/ipfs-log/issues/199
+     - Discussing the "Collaboration Lifecycle" with Marco Oliveira and Pedro Santos: https://github.com/peer-base/peer-base/issues/227
+ - Next:
+     - Keep working on the Identity breakdown, assign people to high-level points outlined there and start working!
+
+#### @andreforsousa
+ - Concluded:
+   - Draft proposal for IDM sitemap with the priorities regarding user-journeys & pages
+   - Helped @satazor with the gist for IDM breakdown
+ - In progress:
+   - Peer-base: Collaboration Lifecycle and UI/UX guidelines with @satazor and @marcooliveira (already uploaded to GitHub)
+   - Kickstart (yesterday) new concept for "IDM"
+   - Follow the current implementation on Discussify animations
+ - Blocked:
+ - Next:
+   - Multiple approaches/concepts for "IDM" UI
+
+#### @dirkmc
+ - Concluded:
+ - In progress:
+   - Fixes in libp2p-switch
+ - Blocked:
+ - Next:
+   - Some changes to how connection management works
+
+#### @marcooliveira
+- Concluded:
+    - Breakdown of peer-base plan (mainly Strategic Product Management Activities and Outbound Activities)
+    - 2019 OKRs
+- In progress:
+    - Porto ProtoSchool chapter
+    - Collaboration Lifecycle draft (current status at https://gist.github.com/marcooliveira/fa5c31aded5b69c9c7deff5b73351729)
+        - Need to discuss with @satazor
+        - Need to discuss with @pgte
+    - Start discussing new iteration on IDM UX & UI with André Cruz & Sousa
+- Blocked:
+- Next:
+    - Events dapp pitch
+    - Define more specific plan for peer-base
+        - resources
+        - milestones
+    - Get approval for:
+        - peer-base plan
+        - Event dapp
+    - Get clarity on what's going to happen with Porto ProtoSchool
+        - Talk to Teri
+        - Talk to Jim Pick
+
+ -------------
+
+## Notes:
+- Jim demoed a peer-base RGA CRDT working in a terminal using nodejs. XState was used as state machine to change between starting, sending, saved, receiving, etc..
+- Pedro mentioned we must rename peer-base because the owner of the peerbase name declined to give it to us. He will make a pool with a few suggestions so that we can decide on the new name.
+- Pedro showed a PR about the WG roadmap in which we all participated in response to Dirk.
+- Arkady tried to reach a 3rd party to adopt our tech but it's too soon. We need to make the stack more stable and reliable first.
+- Adin and Pedro mentioned that we should learn from what textile and bring that knowledge to peer-base. Later on, and if textile agrees, they can replace parts that they built with parts from peer-base.
+
```

Comment:
A few questions:

- Shouldn't all the peer-* apps and building blocks move to the peer-base org? Namely:
  - https://github.com/ipfs-shipyard/peer-pad
  - https://github.com/ipfs-shipyard/workshop-todo-dapp
  - https://github.com/ipfs-shipyard/js-delta-crdts
  - https://github.com/ipfs-shipyard/discussify-styleguide
  - https://github.com/ipfs-shipyard/discussify-browser-extension 
  - https://github.com/ipfs-shipyard/js-ipfs-versidag
  - https://github.com/ipfs-shipyard/discussify-styleguide
  - https://github.com/ipfs-shipyard/js-versidag
  - https://github.com/ipfs-shipyard/pm-discussify
  - https://github.com/ipfs-shipyard/peer-star-infra
  - https://github.com/ipfs-shipyard/pm-peer-pad
  - https://github.com/ipfs-shipyard/pm-idm
  - https://github.com/ipfs-shipyard/peer-star
  - https://github.com/ipfs-shipyard/peer-star-react
  - https://github.com/ipfs-shipyard/peer-star-network-vis-react
  - https://github.com/ipfs-shipyard/peer-star-network-vis
  - https://github.com/ipfs-shipyard/peer-star-peer-color
  - https://github.com/ipfs-shipyard/peer-flipchart
  - https://github.com/ipfs-shipyard/demo-ipfs-todo
  - https://github.com/ipfs-shipyard/peerpad-peer-crdt
  - https://github.com/ipfs-shipyard/dapp-identity-api
  - https://github.com/ipfs-shipyard/dapp-identity-api
  - https://github.com/ipfs-shipyard/peer-pad-core
  - https://github.com/ipfs-shipyard/y-ipfs-connector
  - https://github.com/ipfs-shipyard/peer-crdt-textarea-binding
  - https://github.com/ipfs-shipyard/kipster
  - https://github.com/ipfs-shipyard/peer-identity-demo
  - https://github.com/ipfs-shipyard/peer-identity
  - https://github.com/ipfs-shipyard/peer-crdt
  - https://github.com/ipfs-shipyard/peer-crdt-ipfs
  - https://github.com/ipfs-shipyard/peer-blog
  - https://github.com/ipfs-shipyard/peer-cacl
  - https://github.com/ipfs-shipyard/peer-crdt-example
  - https://github.com/ipfs-shipyard/peer-crdt-bind-codemirror
  - https://github.com/ipfs-shipyard/peer-crdt-platform
  - https://github.com/ipfs-shipyard/ipfs-pubsub-peer-monitor
  - https://github.com/ipfs-shipyard/ipfs-live-db
  - https://github.com/ipfs-shipyard/ipfs-iiif-db
  - https://github.com/ipfs-shipyard/tevere
- Any work planned to rebase the current apps on top of peer-base??

Reply:
Btw, looking at all this repos. If feels it is time to clean up, deprecate and create an onboarding story for all this work. I mean, if I were to build a DApp using IPFS, CRDTs and CACLs, where do I start?

Reply:
Good call, scheduling that in: https://github.com/ipfs/dynamic-data-and-capabilities/issues/64